<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAppointmentDialog</class>
 <widget class="QDialog" name="AddAppointmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddAppointmentDialog</string>
  </property>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fluEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Injection</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>12</month>
     <day>15</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>2024</year>
     <month>12</month>
     <day>15</day>
    </date>
   </property>
   <property name="maximumTime">
    <time>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::Section::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumTime">
    <time>
     <hour>15</hour>
     <minute>59</minute>
     <second>59</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>9</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="timeSpec">
    <enum>Qt::TimeSpec::UTC</enum>
   </property>
   <property name="time">
    <time>
     <hour>9</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QCheckBox" name="newEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>New Patient</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ohipEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
